neo/rawio/intanrawio.py

```diff
@@ -226,25 +229,45 @@ def _parse_header(self):
                     chan_info["gain"],
                     chan_info["offset"],
                     stream_id,
+                    buffer_id,
                 )
             )
             self.native_channel_order[channel_id] = chan_info["native_order"]
         signal_channels = np.array(signal_channels, dtype=_signal_channel_dtype)
 
         stream_ids = np.unique(signal_channels["stream_id"])
         signal_streams = np.zeros(stream_ids.size, dtype=_signal_stream_dtype)
+        
 
         # we need to sort the data because the string of stream_index 10 is mis-sorted.
+        buffer_ids = []
         stream_ids_sorted = sorted([int(stream_id) for stream_id in stream_ids])
         signal_streams["id"] = [str(stream_id) for stream_id in stream_ids_sorted]
-
         for stream_index, stream_id in enumerate(stream_ids_sorted):
             if self.filename.suffix == ".rhd":
                 name = stream_id_to_stream_name_rhd.get(int(stream_id), "")
             else:
                 name = stream_id_to_stream_name_rhs.get(int(stream_id), "")
-
             signal_streams["name"][stream_index] = name
+            # zach I need you help here
```

Comment:
I think we need to say why we don't count these as buffers.

So for header-attached it is because the data is set up in blocks so there is no continous signal
and for one-file-per-channel it is because the stream is split among files.